common/lib/xmodule/xmodule/open_ended_grading_classes/openendedchild.py

```diff
@@ -1,21 +1,30 @@
+"""
+ORA1. Deprecated.
```

Comment:
In your original PR, you reordered the imports in this file. Because you touched Line 1, I had to add a docstring to appease Pylint.

I'm happy to just revert all changes to this file.